File: js/1541.js
var input = require('fs').readFileSync('./dev/stdin', 'utf8')
var lines = input.split('\n')

while (1){

    n = lines.shift()
    if (n.length == 1){
        break
    }
    n = n.split(' ').map(x => parseInt(x))
    
    a = n[0]
    b = n[1]
    c = n[2]
    console.log(`${parseInt(Math.sqrt((a*b*100)/c))}`)
}

File: c/1541.c
#include <stdio.h>
#include <math.h>

int main(){
    long long int i, a, b, c;
    while(1){
        scanf("%lld", &a);
        if (a == 0){
            return 0;
        }
        scanf("%lld %lld", &b, &c);
        
        printf("%lld\n", (long long int) sqrt(a*b*100/c));
    }
}
